src/lib/mongoBackend/mongoRegistrationGet.cpp

```diff
@@ -638,13 +639,52 @@ bool mongoLdRegistrationsGet
     }
   }
 
+  if (ciP->uriParam["type"] != "")
+  {
+    char*                       typeList = (char*) ciP->uriParam["type"].c_str();
+    std::vector<std::string>    typeVec;
+    int                         types;
+    mongo::BSONObjBuilder       bsonInExpression;
+    mongo::BSONArrayBuilder     bsonArray;
+    char*                       details;
+    char                        typeExpanded[256];
+
+    types = stringSplit(typeList, ',', typeVec);
+
+    if (types > 0)
+    {
+      for (int ix = 0; ix < types; ix++){
```

Comment:
Here is the style error.
All brackets go on a separate line, indented with the `for` in this case, like this:

```
      for (int ix = 0; ix < types; ix++)
      {
```

Reply:
Fixed in 818e634